controller/src/controller/services/source/docker.py

```diff
@@ -201,12 +205,13 @@ def menu_images(self) -> list[MenuImage]:
         Returns
         -------
         list of MenuImage
-            All known images.
+            All known images meeting filter criteria.
         """
         registry = self._config.registry
         repository = self._config.repository
         menu_images = []
-        for tag in self._tags.all_tags():
+        filtered_tags = self._tags.apply_policy(self._image_filter)
```

Comment:
Should pass in `datetime.now(tz=UTC)` as the basis for tag ages.

Reply:
I think I prefer leaving this alone and defaulting the time basis in the filter() method to `now()` but I'm certainly willing to be told why I'm wrong.

Reply:
Oh, you said that a couple comments up.